src/createICSSRules.js

```diff
@@ -24,7 +30,7 @@ const createImports = (imports, postcss) => {
   });
 };
 
-const createExports = (exports, postcss) => {
+const createExports = (exports, postcss, asAtRule) => {
```

Comment:
`mode = 'rule' | 'at-rule'`

Reply:
changed to 'atrule' since that's what the postcss `.type` value is